en/micro-integrator/docs/references/connectors/redis-connector/redis-connector-configuration.md

```diff
@@ -0,0 +1,36 @@
+# Setting up the Environment 
+
+The Redis connector allows you to access the Redis commands through the WSO2 EI. Redis stands for remote dictionary server. Redis store /serve that stores data as key-value pairs and this key-value store can be used as a database.
```

Comment:
```suggestion
The Redis connector allows you to access the Redis commands through the WSO2 EI. Redis stands for remote dictionary server. Redis store `/serve` resource that stores data as key-value pairs and this key-value store can be used as a database.
```

Added "resource" here. Check if it is the right term.

Reply:
Updated.